src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -68,14 +76,23 @@ public void parse_allFieldsPresent_success() {
         assertParseSuccess(parser, NAME_DESC_LAB + DEADLINE_DESC_LAB + MODULE_CODE_DESC_HW
                 + MODULE_CODE_DESC_LAB + REMIND_DESC, new AddCommand(expectedAssignmentWithRemind));
 
-        // all fields with remind
+        // TODO JERRY: Find out why an Add Command without priority passes the test case as well eg. expectedAssignment
+        // all compulsory fields with priority
+        assertParseSuccess(parser, NAME_DESC_LAB + DEADLINE_DESC_LAB + MODULE_CODE_DESC_LAB + PRIORITY_DESC,
+                new AddCommand(expectedAssignmentWithPriority));
+
+        // all compulsory fields with remind
```

Comment:
Is it because Priority is an optional field? Does remind have the same problem as well or is it just Priority?

Reply:
Yup remind has the same problem as well. I think it has something to do with how we implement .equals for the Commands in general. We kinda just check whether they're the same type I think